pkg/podexec/pod_command_executor_test.go

```diff
@@ -101,12 +102,34 @@ func TestExecutePodCommandMissingInputs(t *testing.T) {
 				Container: "foo",
 			},
 		},
+		{
+			name:         "hook's container is not overwritten by pod",
+			item:         velerotest.UnstructuredOrDie(`{"kind":"Pod","spec":{"containers":[{"name":"foo"}]}}`).Object,
+			podNamespace: "ns",
+			podName:      "pod",
+			hookName:     "hook",
+			hook: &v1.ExecHook{
+				Container: "bar",
```

Comment:
The `Container` should be `""` to test your change, right?

Reply:
Right. Changed to ""